modules/post/multi/manage/shell_to_meterpreter.rb

```diff
@@ -31,16 +31,29 @@ def initialize(info = {})
         OptAddress.new('LHOST',
           [false, 'IP of host that will receive the connection from the payload.']),
         OptInt.new('LPORT',
-          [false, 'Port for Payload to connect to.', 4433]),
+          [false, 'Port for payload to connect to.', 4433]),
         OptBool.new('HANDLER',
-          [ true, 'Start an Exploit Multi Handler to receive the connection', true])
+          [ true, 'Start an exploit/multi/handler to receive the connection', true])
       ], self.class)
+    register_advanced_options([
+      OptInt.new('HANDLE_TIMEOUT',
+        [false, 'How long to wait for the session to come back', 30]),
+      OptString.new('WIN_TRANSFER',
+        [false, 'Which method to try first to transfer files on a Windows target. Valid values are: POWERSHELL, VBS', 'POWERSHELL']),
+      OptString.new('PAYLOAD_OVERWRITE',
```

Comment:
Can you rename this option and reword its description? It's not clear what it does (sets a different payload by name).